src/compton_tools/python/common_compton.py

```diff
@@ -1,22 +1,23 @@
 #!/usr/bin/env python
-#---------------------------------------*-python-*-------------------------------------------------#
+# --------------------------------------*-python-*------------------------------------------------ #
```

Comment:
Did the linter really want those extra spaces?  I guess I need to update the Emacs template to match.